activiti-cloud-audit-service/activiti-cloud-starter-audit/src/main/resources/config/audit/liquibase/changelog/02-alter.oracle.schema.m10.sql

```diff
@@ -0,0 +1,2 @@
+alter table audit_event
+    add column error_code varchar(255);
```

Comment:
Syntax error for Oracle, it should be `add error_code` instead of `add column error_code`

Reply:
Resolved